app/controllers/admin/organisation_translations_controller.rb

```diff
@@ -10,7 +10,7 @@ def index
 
   def get_layout
     design_system_actions = %w[confirm_destroy]
-    design_system_actions += %w[index] if preview_design_system?(next_release: false)
+    design_system_actions += %w[index edit] if preview_design_system?(next_release: false)
```

Comment:
this needs update to 

```
    design_system_actions += %w[index edit] if preview_design_system?(next_release: false)

```

or you get this on a failed save

<img width="797" alt="image" src="https://github.com/alphagov/whitehall/assets/42515961/ee0231f9-b26d-4e8d-a879-e63639ccb5c8">